internal_packages/keybase/lib/key-manager.cjsx

```diff
@@ -38,7 +38,7 @@ class KeyManager extends React.Component
     popoverTarget = event.target.getBoundingClientRect()
 
     Actions.openPopover(
-      <PassphrasePopover identity={identity} onPopoverDone={ @_exportPopoverDone } />,
+      <PassphrasePopover identity={identity} addresses={identity.addresses} onPopoverDone={ @_exportPopoverDone } />,
```

Comment:
[Is it necessary to pass these separately if they're inside identity already? Honest question, might make sense given the popover's use elsewhere.]